docs/FAQ.md

```diff
@@ -137,6 +137,20 @@ commit, then run `jj restore --from Y --to @-` to restore the parent commit
 to the old state, and `jj restore --from X` to restore the new working-copy
 commit to the new state.
 
+### How do I resume working on an existing change?
+
+There are two ways to resume working on an earlier change: `jj new` then `jj squash`,
+and `jj edit`. The first is the recommended default, but `jj edit` can be useful.
+When you use `jj edit`, you don't get the opportunity to review your draft before it
+is committed to the revision. This can create oversight issues when drafting over
+changes that were considered done. You should avoid using `jj edit` when the
```

Comment:
I'm not sure "can create oversight issues" is clear. Maybe it's better to explain what we mean? Something like the following, perhaps.

> When you `jj edit` a commit, any changes in the working copy will automatically update the commit. That can make it harder to tell what you changed since you resumed working on the commit (though `jj obslog -p` can be helpful). It also makes it harder to abandon the recent changes if you realize you went in the wrong direction.

I think we should also say that there's nothing wrong with `jj edit` if you don't care about those thing, perhaps because your commit was very unfinished.

Again, I think it would also be good to hear from others. I'm pretty sure some people use `jj edit` a lot more liberally than I do.